tests/replicaset/manager-getservers-001.phpt

```diff
@@ -76,7 +76,7 @@ array(3) {
       ["primary"]=>
       string(19) "192.168.112.10:3000"
       ["tags"]=>
-      object(stdClass)#%d (%d) {
+      array(2) {
```

Comment:
I don't see why it has to depend on PHPC-315, this ticket should be able to be resolved alone (by not applying 'document' mapping to the top-level).
Its not going to do that anyway even after PHPC-315.

I don't mind postponing this until PHPC-315 is finished if you feel that is easier then to fix the current 'document' behavior.